roles/adaravis/schema.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+type: enum("adaravis")
+adaravis_genicam_xml_converters_url: str(required=False)
+environment:
+  ENGINEER: str()
+  PREFIX: str()
+  # For camera name, use `/opt/aravis/bin/arv-tool-0.8` to list all cameras
+  # find the one with the IP you assigned to it, and copy the entire entry
+  # without the IP.
```

Comment:
May be worth putting this comment in the example too, so when someone uses the `make ioc` and gets a copy of the example, they know where to get the name from.